code/_onclick/hud/screen_objects.dm

```diff
@@ -495,7 +495,7 @@
 		if("Track With Camera")
 			if(isAI(usr))
 				var/mob/living/silicon/ai/AI = usr
-				var/target_name = input(AI) in AI.trackable_mobs()
+				var/target_name = input(AI) as null|anything in AI.trackable_mobs()
 				AI.ai_camera_track(target_name)
```

Comment:
Have you checked to make sure that this doesn't runtime if you give it a null argument? Otherwise you will want to catch that before trying to call `ai_camera_track`.